Mathlib/Algebra/CharP/Basic.lean

```diff
@@ -170,6 +170,7 @@ end CharZero
 
 namespace Fin
 
+@[stacks 09FS "The characteristic of `F_p` is p."]
```

Comment:
I don't understand why this string isn't just a doc-string.

Reply:
Oh, I think this should be modified to "First part. We don't require p is a prime is mathlib."

Reply:
09FS is 
The characteristic of Fp is p, and that of Q is 0.

This is the copy of the first half of the sentence.

Reply:
Fixed two 09FS tag.